optimum/exporters/openvino/__main__.py

```diff
@@ -409,6 +413,19 @@ def bitnet_load_hook(self, state_dict, prefix, *args, **kwargs):
         if loading_kwargs.get("torch_dtype") == "auto":
             loading_kwargs["torch_dtype"] = dtype
 
+    temporary_directory = None
+    original_output = None
+    # TODO: Remove GPT-OSS workaround when possible
+    quantization_config = None if ov_config is None else ov_config.quantization_config
+    is_generic_quantization = quantization_config and not isinstance(quantization_config, _GPTOSSQuantizationConfig)
+    if is_generic_quantization:
```

Comment:
here the difference is that quantization will be applied after loading the model with an OVModel class is this correct ? Can you add a comment to specify it ?